### PR TITLE
Gerry - search by county, multiple addresses

### PR DESCRIPTION
**Pull request by glaser06**
^^

### CODE REVIEW

**Comment by RebeccaKern** [comment]
Why are my commits from separating out validations in this PR? I don't know if it's a good idea to merge them twice.

File: app/models/address.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class Address < ApplicationRecord
   # before_save :reformat_phone
 
-  belongs_to :location
+  belongs_to :location, required: false
```
Comment (RebeccaKern) [inline app/models/address.rb:4]: In which case would an address not belong to a location?

File: app/controllers/main_controller.rb [hunk 2]
```diff
@@ -46,13 +46,13 @@ def index
...
 
-      item = Item.find(Alias.for_name(qItem.downcase).first.item_id)
+      item = Alias.for_name(qItem.downcase)
 
       if item.blank?
         @errors += "Could not find #{params[:item]}"
         return
       end
-      @item = item
+      @item = Item.find(item.first.item_id)
```
Comment (RebeccaKern) [inline app/controllers/main_controller.rb:55]: Does this mean zipcode is still required?

File: app/models/address.rb (hunk 1 above)
Comment (glaser06) [reply app/models/address.rb:4]: when you need to add one for a nested form. it's similar to how the item_location was done.

**Pull request closed without merging**